apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -1,24 +1,56 @@
 // @flow
 
 import * as React from 'react';
-import { FlatList } from 'react-native';
+import { FlatList, View } from 'react-native';
 import { formatPrice } from '@kiwicom/margarita-utils';
 import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { TouchableWithoutFeedback } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { StyleSheet, Hoverable } from '@kiwicom/universal-components';
 
 import RenderTripSectorItem from './RenderTripSectorItem';
 import ItineraryCardWrapper from './ItineraryCardWrapper';
+import ItineraryDetail from './itineraryDetail/ItineraryDetail';
 import type { ItineraryCard as ItineraryCardType } from './__generated__/ItineraryCard.graphql';
 import type { RenderTripSectorItem as RenderTripSectorItemType } from './__generated__/RenderTripSectorItem.graphql';
 
 type Props = {|
   +data: ItineraryCardType,
 |};
 
+type State = {|
+  detailOpen: boolean,
```

Comment:
What about rename it to  `detailOpened` or `isDetailOpened` (`isHovered`)?

Reply:
Sure, renamed to `detailOpened` to make it more unified.